DKHelper/NSString+DKHelper.m

```diff
@@ -111,13 +111,13 @@ - (BOOL)isEmail {
 }
 
 - (BOOL)isNumeric {
-	NSString *regex = @"^[0-9]+[.]?[0-9]*$";
+	NSString *regex = @"^[-]*[0-9]+(.[0-9]+)?$";
 	NSPredicate *predicate = [NSPredicate predicateWithFormat:@"SELF MATCHES %@", regex];
 	return [predicate evaluateWithObject:self];
 }
 
 - (BOOL)isAlphaNumeric {
-    NSString *regex = @"[A-Z0-9a-z_]+";
+    NSString *regex = @"[A-Z0-9a-z]+";
```

Comment:
How about ```[A-Z0-9a-z.\s]+```?
This way you would also catch whitespaces and positive floats ```3.2```. Problem: You'd miss ```-3.2```

Maybe combine the ```isNumeric``` return value with ```[\w\s.]+```